Formula/xserver.rb

```diff
@@ -0,0 +1,114 @@
+class Xserver < Formula
+  desc "X Window System display server"
+  homepage "https://www.x.org"
+  url "https://gitlab.freedesktop.org/xorg/xserver.git",
+    tag:      "xorg-server-21.1.3",
+    revision: "85397cc2efe8fa73461cd21afe700829b2eca768"
+  license all_of: ["MIT", "APSL-2.0"]
+
+  depends_on "font-util"   => :build
+  depends_on "libxkbfile"  => :build
+  depends_on "meson"       => :build
+  depends_on "ninja"       => :build
+  depends_on "pkg-config"  => :build
+  depends_on "util-macros" => :build
+  depends_on "xorgproto"   => :build
+  depends_on "xtrans"      => :build
+
+  depends_on "libapplewm"
```

Comment:
Should be macOS-only dependency.

Reply:
Yes, I put it here because I don't know how to fix the test on linux. 😓

Reply:
Although the Linux CI environment's `DISPLAY` is often an issue with X11 tests, `xserver` actually provides `Xvfb` which can emulate a dumb framebuffer and can actually be used to run tests on headless machines.

Also, maintainers can help fix any other Linux issues.

Reply:
I will try it later.